process/slack_etiquette.md

```diff
@@ -0,0 +1,64 @@
+
+### :star2: Don’t “ask to ask”
+Newcomers very commonly ask about asking a question in one of a few ways.
+.
+_**“Does anyone know about [X tool or library]?”**_
+The answer is probably **“yes,”** but nobody can help you until you state what your problem **is**. All too often by the time somebody asks what the **real** question is, the person who needed help has disconnected. 
+.
+_**Please be specific with your questions so all of us can help you even when you aren't online.**_ Share a code snippet, a link to where you're seeing the issue, a description of what is confusing you, and specifics on what you've tried so far.  More often than not, it also helps to paste your error logs into a snippet or text file (instructions here on how to do that in Slack) to accompany your question/issue.
+.
+_**“Is it okay if I ask about [X tool or library] here?”**_
+We have a dizzying array of channels, which we’re aware of and for which we apologize. We have utility channels such as #announcements, #general, #advice, etc.  All our channels are listed alphabetically. 
+.
+Instead of asking, browse the Channels list for any that seem like obvious places to ask your question. If you don’t see any topics or Channels that fit, don’t hesitate to post it in #general or #advice. If there’s a better place for it, one of the moderators or other helpful users will suggest it, but don't worry - off topic messages aren’t going to get you banned.
+
+### :star2: Don’t use `@here`, `@channel`, or ping / DM members unless it's directly relevant
+It’s very tempting to try to get more attention to your question by @-mentioning one of the high profile (or recently active) members, but please don’t. They may not actually be online, they may not be able to help, and they may be in a completely different timezone–nobody likes push notifications at 3am from an impatient stranger.
+.
+Similarly, don’t DM other members without asking first. All of the same problems as @-mentioning apply, and private conversations can’t help anyone else. Your questions are likely not unique, and other people can learn from them when they’re asked in a public channel.
+
+### :star2: Be helpful
+We are a pay-it-forward community. If someone here has helped you, please give back by helping someone else!
+.
+It doesn’t matter if you can’t answer a question in a brief sentence, or if you get pulled away before you can type out further replies. If a question isn’t clear, ask a clarifying question. If you remember something you read that seemed related, send a link. A lot of people who are stuck will be fine if you can’t provide the solution, they just need another place to look.
+
+### :star2: Be polite
+All the CodeBuddies here deserve your respect, so be polite. Not “retail employee” polite......but don’t fan flamewars, get sucked into lose-lose arguments, or other internet classics.
```

Comment:
*Edit:*

Strike to `PyLadies`. Additionally reference this: `As a fiscal sponsoree of the Python Software Foundation, all PyLadies spaces are governed by the PSF Code of Conduct. You can read the Code of Conduct [here](https://www.python.org/psf/conduct/).`